tests/cstor/pool/reconciliation/reconciliation_test.go

```diff
@@ -300,3 +301,92 @@ var _ = Describe("RAIDZ2 SPARSE SPC", func() {
 	})
 
 })
+
+var _ = Describe("SPC POOL POD DELETION", func() {
+	When("We apply sparse-striped-auto spc yaml with maxPool count equal to 1"+
+		"on a k8s cluster having at least one capable node", func() {
+		It("pool resources count should be 1 with no error and healthy status", func() {
+			namespace := string(artifacts.OpenebsNamespace)
+			builtSpcObj := spc.NewBuilder().
+				WithGenerateName(spcName).
+				WithDiskType(string(apis.TypeSparseCPV)).
+				WithMaxPool(1).
+				WithOverProvisioning(false).
+				WithPoolType(string(apis.PoolTypeStripedCPV)).
+				Build()
+			spcObj = builtSpcObj.Object
+
+			// Create a storage pool claim
+			_, err := ops.SPCClient.Create(spcObj)
+			Expect(err).To(BeNil())
+			cspCount := ops.GetHealthyCSPCount(spcObj.Name, 1)
+			Expect(cspCount).To(Equal(1))
+
+			// get pool pod corresponding to above spc
+			poolPodList, err := ops.PodClient.WithNamespace(namespace).
+				List(metav1.ListOptions{
+					LabelSelector: "openebs.io/storage-pool-claim=" + spcObj.Name,
+				},
+				)
+			Expect(err).To(BeNil(), "failed to get list of pool pods")
+
+			// verify pod status
+			status := ops.IsPodRunningEventually(namespace, poolPodList.Items[0].Name)
+			Expect(status).To(Equal(true), "while checking status of pool pod {%s}", poolPodList.Items[0].Name)
```

Comment:
can we have this like to make it more readable ?
```go
Expect(status).To(
    Equal(true), 
    "while checking status of pool pod {%s}", 
    poolPodList.Items[0].Name,
)
```
similarly all other places

Reply:
Done